_posts/2017-08-16-node_starter_guide.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "Starters Guide to Node.js"
+date: 2017-08-16 00:00:00 +530
+author: Hritvik Taneja
+category:
+tags:
+- node
+- Node.js
+- npm
+---
+## `console.log('Hello World')`
+
+I assume that most of you are here to learn about Node.js. So, lets get started!!
+
+## What is Node.js ?
+
+**If you go by docs:** [Node.js®](https://nodejs.org/en/) is a JavaScript runtime built on [Chrome's V8 JavaScript engine](https://developers.google.com/v8/). Node.js uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node.js' package ecosystem, [npm](https://www.npmjs.com/), is the largest ecosystem of open source libraries in the world.
+
+In simple words Node.js is server side programming environment which helps us to create APIs which can be consumed by Mobile devices, Websites etc..
+
+In 2009 Ryan Dahl released Node.js to run JavaScript on servers such as Linux and IIS Windows servers.
+
+## Why should one learn Node.js ?
+
+JavaScript is undoubtedly one of the most popular programming languages out there today, and for good reason. It can easily be run in your browser, on a server, on your desktop, or even on your phone as an app.
+
+Before Node.js, JavaScript was only client side language meaning it can run only on browsers such as chrome, Mozilla etc.
+
+You can find loads of stuff on the internet which will persuade you to learn Javascript and Node.js, but one reason which I find the most compelling is that you can pursue both frontend and backend development if you learn the above stated.
```

Comment:
Link them, and create a small summary table like APwhitehat has done on his RoR post. Listing "cons" along with pros is also a plus